gax/operations_stub_unittest.cc

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google Inc.  All rights reserved
```

Comment:
nit: this is not the format recommended by go/releasing/preparing#Apache-header

Reply:
Oy. I've been using the same boilerplate for all files. Nice catch.